guides/common/modules/proc_removing-packages-from-a-host.adoc

```diff
@@ -0,0 +1,12 @@
+[id="removing-packages-from-a-host_{context}"]
+= Removing Packages from a Host
+
+You can remove packages from a host in the {ProjectWebUI}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the name of the host you want to modify.
+. Go to the *Content* tab, then click the *Packages* tab.
```

Comment:
We haven't been using subtabs up until now but I think I could be a good way to differentiate between the levels of inclusion. @maximiliankolb @Lennonka @mjivraja what do you think?

```suggestion
. On the *Content* tab, navigate to the *Packages* subtab.
```

Reply:
If there's a tab on a tab, then I'm OK with "subtab" if used consitently. What does IBM Style Guide say about that?

Reply:
I have not found anything speaking for or against it, but I think it would be logical to use in our case.

Reply:
I just had a discussion with style-guide people. It is not recommended to use `subtab` at this level (unless it's officially documented) citing RH Supplementary & IBM Style Guide. 

We should revert it to just `tab`

Reply:
Feel free to apply [Lena's wording](https://github.com/theforeman/foreman-documentation/pull/2117#discussion_r1162806396), then.